packages/react-server/src/ReactFizzServer.js

```diff
@@ -2261,6 +2262,7 @@ function renderElement(
       task.keyPath = prevKeyPath;
       return;
     }
+    case REACT_ACTIVITY_TYPE:
```

Comment:
@sebmarkbage this might not be correct - to match the client it should render an Offscreen element, but I'm not sure how to do that in Fizz. Without that, I think this might cause a hydration error if it's rendered in mode `visible`?

Reply:
Offscreen is an implementation detail of Fiber and it will never be noticeable in the HTML so it can't hydrate.

Basically Fizz doesn't need to know about the concept of Offscreen. It doesn't need the same type of code sharing between Suspense and Activity that Fiber needs to manage the complexity. So it can be just forked paths.

In other words, you can just drop `REACT_OFFSCREEN_TYPE` here and rename it to Activity.

Reply:
Nice, makes sense. Done!